src/calendar/day/basic/index.js

```diff
@@ -92,26 +93,29 @@ class Day extends Component {
     }
 
     return (
-      <TouchableOpacity
-        testID={this.props.testID}
-        style={containerStyle}
-        onPress={this.onDayPress}
-        onLongPress={this.onDayLongPress}
-        activeOpacity={activeOpacity}
-        disabled={shouldDisableTouchEvent}
-        accessibilityRole={isDisabled ? undefined : 'button'}
-        accessibilityLabel={this.props.accessibilityLabel}
-      >
-        <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
-        <Dot
-          theme={theme}
-          isMarked={marked}
-          dotColor={dotColor}
-          isSelected={selected}
-          isToday={isToday}
-          isDisabled={isDisabled}
-        />
-      </TouchableOpacity>
+      <View>
+        <TouchableOpacity
+          testID={this.props.testID}
+          style={containerStyle}
+          onPress={this.onDayPress}
+          onLongPress={this.onDayLongPress}
+          activeOpacity={activeOpacity}
+          disabled={shouldDisableTouchEvent}
+          accessibilityRole={isDisabled ? undefined : 'button'}
+          accessibilityLabel={this.props.accessibilityLabel}
+        >
+          <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
+          <Dot
+            theme={theme}
+            isMarked={marked}
+            dotColor={dotColor}
+            isSelected={selected}
+            isToday={isToday}
+            isDisabled={isDisabled}
+          />
+        </TouchableOpacity>
+        {underView}
```

Comment:
It would be better to pass a function instead of an element here.
{!!renderUnderView && renderUnderView()}

Also, you should add this new prop to `propTypes`

Reply:
Changed